inspirehep/modules/workflows/templates/inspire_workflows/index.html

```diff
@@ -88,6 +88,16 @@ <h3 class="text-center header-text">
 
         <holding-pen-dashboard-item
           template="{{url_for('static', filename='node_modules/inspirehep-search-js/dist/templates/inspirehep-holdingpen-js/dashboard_stat_block.html')}}"
+          filter_string="is-update=true&workflow_name=HEP"
```

Comment:
The symmetric should be reflected also in the other box: `is-update=false` into the arXiv box.